addons/test_mail/tests/__init__.py

```diff
@@ -20,3 +20,4 @@
 from . import test_performance
 from . import test_res_users
 from . import test_odoobot
+from . import test_read_progress_bar
```

Comment:
Please put it into test_mail_activity test file. We try to keep tests organized by section.